tests/unittest/aws_test.py

```diff
@@ -383,5 +450,91 @@ def testSnapshot(self, mock_ec2_api):
     self.assertTrue(snapshot.name.startswith('my-snapshot'))
 
 
+class AWSTest(unittest.TestCase):
+  """Test the aws.py public methods."""
+
+  @mock.patch('libcloudforensics.aws.AWSVolume.Snapshot')
+  @mock.patch('libcloudforensics.aws.AWSAccount.GetVolumeById')
+  @mock.patch('libcloudforensics.aws.AWSAccount.GetAccountInformation')
+  @mock.patch('libcloudforensics.aws.AWSAccount.ClientApi')
```

Comment:
You can make use of the `assert_called_*` functions on the mock objects: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called

This allows you to make sure that the functions you mock are being called with the expected arguments.

As a general comment I think that other tests could also benefit from these (especially in `testGetOrCreateAnalysisVm`)

Reply:
I added a few more statements on these; and opened #73 to keep track.